nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/pandar_qt128.hpp

```diff
@@ -45,20 +47,75 @@ struct TailQT128C2X
 
 struct FunctionalSafetyQT128C2X
 {
+  static constexpr uint64_t update_cycle_ns = 5'000'000;
+
+  enum class LidarState : uint8_t {
+    Initialization = 0,
```

Comment:
Without checking how all other modules work, I think mostly we use upper snake case for enumerations? Or does it get a bit shouty?

Reply:
[Autoware](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/#use-upper-camel-case-for-enum-names-required-partially-automated) apparently suggests CamelCase. I only learned about that recently, so I'm not sure if we should aim for it to fit in with the rest of Nebula, or if we should gradually aim to achieve Autoware style compliance.

Reply:
ROS uses upper snake which is why Nebula did originally. I also wasn't aware about Autoware.

Reply:
Addressed in 9d5e2c5.